.storybook/manager-head.html

```diff
@@ -0,0 +1,34 @@
+<style>
+  .sidebar-item {
+    padding-bottom: 6px !important;
+    padding-top: 6px !important;
+  }
+
+  .sidebar-item svg {
+    color: rgba(0,0,0,0.9) !important;
+  }
+
+  .sidebar-item[data-selected="true"] {
+    color: rgba(0,0,0,0.9) !important;
+    font-weight: 500 !important;
+  }
+
+  .sidebar-item[data-selected="true"] svg {
+    color: rgba(0,0,0,0.9) !important;
+  }
+
+  .sidebar-item[data-selected="false"]:hover,
+  .sidebar-item[data-nodetype="group"]:hover,
+  .sidebar-item[data-nodetype="component"]:hover {
+    background: #007E45 !important;
+    color:  white !important;
+  }
+
+  .search-result-item--label mark {
+    color:  #0060a7 !important;
+  }
+
+  .css-1va4tas {
```

Comment:
This class will be re-generated when we upgrade storybook. Is there another class you can use?

Reply:
I wasn't sure if that was the case as I rebuilt and it seemed to stay the same, I will double check, otherwise I will remove this line.

Reply:
I think it's a guid based on the styles of the component. So It'll only change if we upgrade storybook, and, the styles change in storybook itself.

Reply:
Think Jamie is correct about this, I haven't seen anything in the docs that offers an alternate way of achieving this. Maybe a little bit less brittle (but still less than ideal) approach would be to target `[role="main"]` or just not have the box shadow at all for now 🤷 

PS you can also remove `!important` here if you do